JetMETCorrections/Modules/src/SealModule.cc

```diff
@@ -58,6 +59,9 @@ DEFINE_FWK_EVENTSETUP_MODULE(JetCorrectionESChain);
 DEFINE_FWK_EVENTSETUP_MODULE(QGLikelihoodESProducer);
 DEFINE_FWK_EVENTSETUP_MODULE(QGLikelihoodSystematicsESProducer);
 
+DEFINE_FWK_EVENTSETUP_MODULE(JetResolutionESProducer);
+DEFINE_FWK_EVENTSETUP_MODULE(JetResolutionScaleFactorESProducer);
+
```

Comment:
Request for bug fix: please move:
https://github.com/blinkseb/cmssw/blob/jer_gt_rebased/JetMETCorrections/Modules/src/SealModule.cc#L32
https://github.com/blinkseb/cmssw/blob/jer_gt_rebased/JetMETCorrections/Modules/src/SealModule.cc#L33
(i.e. `DataProxy` registration in the `ESSource` for JEC) into `CondCore/JetMETPlugins/src/plugins.cc`